umap/static/umap/css/bar.css

```diff
@@ -219,7 +219,7 @@
     display: none;
     top: calc(var(--current-header-height) + var(--panel-gutter));
     flex-direction: column;
-    z-index: var(--zindex-panels);
+    z-index: calc(var(--zindex-panels) - 1);
```

Comment:
The edit bar being declared below in the DOM, it stays above the panels otherwise (which leads to inaccessible close/extend buttons).